### ISSUE TITLE
generate_workspace failed when parsing com.google.cloud:google-cloud:0.3.0

### ISSUE THREAD

**Issue by meteorcloudy** [bazelbuild/bazel#1794]
```
$  bazel run //src/tools/generate_workspace -- -a com.google.cloud:google-cloud:0.3.0
INFO: Analysed target //src/tools/generate_workspace:generate_workspace.
INFO: Found 1 target...
Target //src/tools/generate_workspace:generate_workspace up-to-date:
  C:/temp/Bazel/Hpl65qab/execroot/bazel_gerrit/bazel-out/local-fastbuild/bin/src/tools/generate_workspace/generate_workspace.jar
  C:/temp/Bazel/Hpl65qab/execroot/bazel_gerrit/bazel-out/local-fastbuild/bin/src/tools/generate_workspace/generate_workspace
INFO: Elapsed time: 2.857s, Critical Path: 0.93s
INFO: Build completed successfully, 4 total actions

INFO: Running command line: C:/temp/Bazel/Hpl65qab/execroot/bazel_gerrit/bazel-out/local-fastbuild/bin/src/tools/generate_workspace/generate_workspace -a com.google.cloud:google-cloud:0.3.0
ERROR: Could not resolve dependency io.grpc:grpc-core:[0.15.0]: Could not find any repositories that knew how to resolve io.grpc:grpc-core:[0.15.0] (checked https://oss.sonatype.org/content/repositories/snapshots, http://people.apache.org/repo/m2-snapshot-repository, https://repo1.maven.org/maven2/)
ERROR: Could not resolve dependency io.netty:netty-codec-http2:[4.1.1.Final]: Could not find any repositories that knew how to resolve io.netty:netty-codec-http2:[4.1.1.Final] (checked https://oss.sonatype.org/content/repositories/snapshots, http://people.apache.org/repo/m2-snapshot-repository, https://repo1.maven.org/maven2/)
Wrote:
C:\Users\pcloudy\AppData\Local\Temp\1474294558745-0\WORKSPACE
C:\Users\pcloudy\AppData\Local\Temp\1474294558745-0\BUILD
```

This is a blocker for @vuryleo who is using Bazel.

**Comment by meteorcloudy** [comment]
Looks like the version number should not contain []
`[0.15.0]` -> `0.15.0`

**Comment by buchgr** [comment]
The bracket syntax has special semantics in maven. It means that a project really needs this particular version of the dependency. Else, if there are multiple competing versions of a dependency, maven is free to choose which version to pick.

**Comment by kchodorow** [comment]
Fixed by db58128.
